### PR TITLE
Victor/NFT-546 Add cursor to Sylvester subgraph 

### PR DESCRIPTION
**Pull request by victonesha**
Added the cursor similar to the avalanche subgraph implementation. At some point in time after we get this working on prod we should look into merging the tests & enhancements from the other branches.

### CODE REVIEW

File: mappings/core.ts [hunk 1]
```diff
@@ -24,10 +19,17 @@ export function handleLend(event: Lend): void {
...
+    lender = new User(lentParams.lenderAddress.toHexString());
+    counter.user = counter.user + 1;
+    lender.cursor = counter.user;
+  }
   lending.user = lender.id;
-  lrc.lending = lrc.lending.plus(BigInt.fromI32(1));
-  lrc.save();
+  counter.lending = counter.lending + 1;
+  lending.cursor = counter.lending;
+
```
Comment (nazariyv) [inline mappings/core.ts:32]: you need to save counter here or it will not be persisted.

File: mappings/core.ts [hunk 2]
```diff
@@ -36,58 +38,60 @@ export function handleRent(event: Rent): void {
...
+    counter.user = counter.user + 1;
+    renter.cursor = counter.user;
+  }
   renting.user = renter.id;
-  lrc.renting = lrc.renting.plus(BigInt.fromI32(1));
-  lrc.save();
+  counter.renting = counter.renting + 1;
+  renting.cursor = counter.renting;
+
   lending.save();
```
Comment (nazariyv) [inline mappings/core.ts:65]: you need to save counter here or it will not be persisted.

**Review by nazariyv — CHANGES_REQUESTED** [review]

File: mappings/core.ts (hunk 1 above)
Comment (bauti-defi) [reply mappings/core.ts:32]: fixed.

File: mappings/core.ts (hunk 2 above)
Comment (bauti-defi) [reply mappings/core.ts:65]: fixed.

**Review by nazariyv — APPROVED** [review]

**Pull request merged**